src/einheit/utils.nim

```diff
@@ -45,9 +45,9 @@ else:
       ws_xpixel: cushort
       ws_ypixel: cushort
 
-  let
-    IOC_OUT = 0x40000000
-    IOCPARM_MASK = 0x1fff
+  # let
+    # IOC_OUT = 0x40000000
+    # IOCPARM_MASK = 0x1fff
```

Comment:
These can just be deleted. I put them in here because I was going to use them but forgot to remove them :)